### ISSUE TITLE
Adding a static DNS server to a connection

### ISSUE THREAD

**Issue by ivanstegic** [ivanstegic/menu-bar-dns#11]
### Requirements for v0.0.5

1. It is possible to add a static IP address to any network connection
2. If there is an existing DHCP assigned IP, the static IP replaces it

![v005](https://user-images.githubusercontent.com/509217/104872652-0b602380-5914-11eb-85c6-9043efea2d06.png)

**Comment by munirahmed007** [comment]
Question:
we display DNS IP address for Ethernet on NSMenu collectively from all Ethernet Interfaces.
When user adds a new IP address, do we add it to all Ethernet interfaces? @ivanstegic

**Comment by ivanstegic** [comment]
@munirahmed007 no, the added IP address only gets added to the network connection that was selected when (+) was clicked.

**Comment by ivanstegic** [comment]
As in this example above, it only gets added to the **Ethernet** connection, but not to the **WiFi** connection. It just so happens that the **WiFi** connection already has the same static DNS IP added. Just a coincidence. Does that make sense?

**Comment by munirahmed007** [comment]
@ivanstegic 
I understand, sorry that I didn't make myself clear.
Even for Ethernet, we can have multiple ethernet services/interfaces installed. Like in my computer, I've two USB ethernet interfaces leading to different LANs. so If I'm correct, we will add DNS IP for both of these Ethernet interfaces. I hope I made my question clear this time. 

Similarly one can have multiple Wi-Fi interfaces installed on laptop.

**Comment by ivanstegic** [comment]
No, I think you have it wrong.

If you add one IP address as a new static DNS server, that single IP should only ever be assigned to **one active connection** not all of them. So, I should be able to hover over one of your two USB Ethernet connections, press the plus (+) sign next to that one, and add the static DNS server to only that connection. You should NOT assign that IP to the other USB Ethernet device, nor should you assign it to any other device or connection. 

Is that clear?

**Comment by munirahmed007** [comment]
Ok. Now it's clear. However, attached snapshots were misleading, according to which, (+) was supposed to appear on `Ethernet` MenuItem. Perhaps we should change the snapshot/description based on this conversation.

**Comment by ivanstegic** [comment]
Let's see what the PR looks like, and we can update accordingly.
